dataprofiler/profilers/order_column_profile.py

```diff
@@ -270,6 +270,22 @@ def report(self, remove_disabled_flag: bool = False) -> dict:
         """
         return self.profile
 
+    @classmethod
+    def load_from_dict(cls, data):
+        """
+        Parse attribute from json dictionary into self.
+
+        :param data: dictionary with attributes and values.
+        :type data: dict[string, Any]
+
+        :return: Profiler with attributes populated.
+        :rtype: CategoricalColumn
+        """
+        # This is an ambiguous call to super classes.
+        profile = super().load_from_dict(data)
+
+        return profile
```

Comment:
```suggestion
        return super().load_from_dict(data)
```
I see benefits to either way, just putting another option out there

Reply:
cleaner +1

Reply:
@ksneab7

Reply:
the other implementation do operations on `profile`, but in this case we don't .... so no need I don't think to set it to a variable